src/app/portlets/dot-edit-page/layout/dot-edit-layout-grid/dot-edit-layout-grid.component.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { ComponentFixture } from '@angular/core/testing';
+
+import { NgGridModule } from 'angular2-grid';
+import { Input, Component } from '@angular/core';
+
+import { DotEditLayoutGridComponent } from './dot-edit-layout-grid.component';
+import { DOTTestBed } from '../../../../test/dot-test-bed';
+
+
+@Component({
+    selector: 'action-button',
+    template: ''
+})
+
+class TestActionButtonComponent {
```

Comment:
Mauybe I'm missing somehing but I don't see this class it's being used anywhere in this test.

Reply:
Is part of the template, so if I don't added the test will not run. is to mock the action button.